client/src/components/common/applicant-info-1.vue

```diff
@@ -589,7 +589,7 @@ export default class ApplicantInfo1 extends NameRequestMixin {
 
   @Watch('isValid')
   onValidChanged (val: boolean) {
-    if (val) {
+    if (val && this.$el?.querySelector) {
```

Comment:
I added this in several places. Would you believe `this.$el` is sometimes undefined? It's true!

(It doesn't really matter if the code in this block is run -- it's only for better diagnostics.